airflow/dags/dags.py

```diff
@@ -3,7 +3,7 @@
 
 from ewah.dag_factories import dags_from_yml_file
 
-for dag in dags_from_yml_file(conf.get("core", "dags_folder") + os.sep + "dags.yml"):
+for dag in dags_from_yml_file(conf.get("core", "dags_folder") + os.sep + "dags_g.yml"):
```

Comment:
Request (blocking): Please don't push this change; this is purely to make your local development work.

Ideally, you could think of a way to abstract the issue such that one can locally configure what yaml files to load.

Reply:
That's exactly what was my comment about. 

This means I have to remove the change dag_g.yml and go back to dags.yml, meanwhile I have to delete the dags_g.yml file itself from the repo. 

But, if I do this, I won't be able to test the change without affecfting production, right?